recipes/daw_utf_range/all/conandata.yml

```diff
@@ -11,3 +14,9 @@ sources:
   "2.2.0":
     url: "https://github.com/beached/utf_range/archive/v2.2.0.tar.gz"
     sha256: "00f60360736062403c8a7a94dd07c750366958f20d1864578faecf2e09d84265"
+daw_headers_mapping:
```

Comment:
@toge with this change, new versions of the library do not need to change the revisions of old ones - we can apply this pattern to other libraries that have the same "1:1" version correspondance between eachother from now on! :)

Reply:
This is because we use the `trim_conandata` hook ([Docs here](https://docs.conan.io/2/reference/tools/files/basic.html#conan.tools.files.conandata.trim_conandata))

Reply:
@AbrilRBS 
Thank you for modification!
I try to use the feature.